vhive.go

```diff
@@ -81,6 +82,7 @@ func main() {
 	isLazyMode = flag.Bool("lazy", false, "Enable lazy serving mode when UPFs are enabled")
 	criSock = flag.String("criSock", "/etc/vhive-cri/vhive-cri.sock", "Socket address for CRI service")
 	hostIface = flag.String("hostIface", "", "Host net-interface for the VMs to bind to for internet access")
+	netPoolSize = flag.Int("netPoolSize", 10, "Amount of network configs to preallocate in a pool")
```

Comment:
yep, 10 is the right number. should be set to 10 elsewhere as well.

Reply:
Fixed.